scripts/translate_locales.py

```diff
@@ -0,0 +1,33 @@
+
+# A script to generate a template which will be used to localize the supported locale name
+# For more inofrmation see https://bugzil.la/859499#c11
+
+from django.conf import settings
+
+from jinja2 import Environment
+
+
+template_string = ("""
+
+This template is automatically generated by scripts/translate_locales.py to make the languages name localizable. Do not edit it manually
+Background: https://bugzil.la/859499#c11
+
+{% for lang in languages %}
+   {{ '{{ _("' + lang + '") }}' }}
+{% endfor %}
+
+""")
+
+
+LANGUAGES = sorted([lang_info.english.encode('ascii', 'ignore')
+                    for lang_code, lang_info in settings.LOCALES.items()
+                    if lang_code in settings.MDN_LANGUAGES])
+
+string = Environment().from_string(template_string).render(languages=LANGUAGES)
```

Comment:
Jinja is probably overkill for this usage - we don't need the HTML escaping, template caching, etc. Plain Python string construction should work.  Something like:

```
    string = (u'''\
This template is automatically generated by scripts/translate_locales.py to make the languages name localizable. Do not edit it manually
Background: https://bugzil.la/859499#c11

  %s
''') % '\n  '.join(LANGUAGES)
```

This assumes that ``LANGUAGES`` contains a list of unicode strings, rather than ``encode`` to switch to bytes.  Then, when you write to the file, you can encode the whole thing in utf-8.